sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsOptions.cs

```diff
@@ -67,7 +67,7 @@ public int MaxDequeueCount
                     throw new ArgumentException("MaxDequeueCount must not be less than 1.", nameof(value));
```

Comment:
Need to update this error message as well. Use nameof in the error string. If there isn't a unit test for this, perhaps we should add one.